prime-router/metadata/fhir_mapping/hl7/segments/PID/Provenance-Patient.yml

```diff
@@ -0,0 +1,60 @@
+resourceType: Provenance
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+target:
+  valueOf: datatype/Reference
+  generateList: true
+  expressionType: resource
+  specs: $Patient
+
+recorded:
```

Comment:
Can we add a tech debt ticket that would allow us to set a transform start variable that would be available globally?  I think technically our `recorded` values might differ by a very tiny amount

Reply:
It's only down to second precision, so I haven't been able to hit that, but it does seem possible. I can see about setting the transformation time as a variable at the base transform level and using that in each spot, it might only be a couple lines and save us the ticket.

Reply:
Didn't work, so opened a ticket [here](https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/13134)